package/greenpak-programmer/greenpak-programmer.hash

```diff
@@ -0,0 +1,2 @@
+# From https://github.com/addiva-elektronik/greenpak-programmer/releases/download/v1.0/greenpak-programmer-1.0.tar.gz.sha256
+sha256  eb7c8347fef8764f0f31f645570a6ab2842aabecff3de83c1a2dcd79f1e4bdcc  greenpak-programmer-1.0.tar.gz
```

Comment:
When we run `make legal-info` to generate an SBOM for customers we'd need a sha256 listing also for the LICENSE file.  I'm not sure what happens if there is no LICENSE file in the package ... could you check?